chore(random): adding randomness and stopOnFailure fields (#478)
litmuschaos/litmus-docs: docs/docker-service-kill.md

  </tr>
</table>

> <strong>Note:</strong> The experiment currently support for GKE cluster with Ubuntu with Docker (Ubuntu) image. The Container-Optimised OS with Containerd needs to have different pre and post chaos check that will be added in subsequent releases.

Reviewer: ```suggestion
<strong>Note:</strong> The experiment is currently supported on GKE built on nodes with docker runtime. The Container-Optimised OS with Containerd needs to have different pre and post chaos checks that will be added in subsequent releases.
```
